src/file-uploader/file-uploader.stories.ts

```diff
@@ -46,6 +47,7 @@ class FileUploaderStory {
 	@Input() title;
 	@Input() description;
 	@Input() buttonText;
+	@Input() buttonType = "primary";
```

Comment:
It could use the typing.
```suggestion
	@Input() buttonType: "primary" | "secondary" | "tertiary" | "ghost" | "danger" = "primary";
```